contracts/src/v0.8/keystone/interfaces/IReceiver.sol

```diff
@@ -1,8 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import {IERC165} from "../../vendor/openzeppelin-solidity/v5.0.2/contracts/utils/introspection/IERC165.sol";
+
 /// @title IReceiver - receives keystone reports
-interface IReceiver {
+interface IReceiver is IERC165 {
```

Comment:
please add a comment that it's important to support IReceiver or no reports would be sent